astraconnector_operator.yaml

```diff
@@ -462,7 +461,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: netapp/astra-connector-operator:1.0.202209131709
+        image: netapp/astra-connector-operator:0.0.1
```

Comment:
is this intentional?

Reply:
no, not sure how that happened.  In fact, it should go to 2.0

Reply:
the regenerate script sets that back to 0.0.1

Reply:
fixed now